api/envoy/api/v2/listener.proto

```diff
@@ -236,4 +237,8 @@ message Listener {
   // This issue was fixed by `tcp: Avoid TCP syncookie rejected by SO_REUSEPORT socket
   // <https://github.com/torvalds/linux/commit/40a1227ea845a37ab197dd1caffb60b047fa36b1>`_.
   bool reuse_port = 21;
+
+  // Configuration for :ref:`access logs <arch_overview_access_logs>`
```

Comment:
Can you add some docs with example log messages, etc?

Reply:
Should I do that in general docs or the API? It's pretty much identical to tcp_proxy logging capability.

Reply:
This needs a release note. Also you can please add some text to https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/access_logging that describes this capability (and potentially fleshes out some of the other current uses)?

Reply:
Done.